docs/docs/Contributing/contributing-how-to-contribute.md

```diff
@@ -4,157 +4,110 @@ sidebar_position: 1
 slug: /contributing-how-to-contribute
 ---
 
+# How to contribute to Langflow
 
+This guide is intended to help you get started contributing to Langflow.
+As an open-source project in a rapidly developing field, we are extremely open
+to contributions, whether it be in the form of a new feature, improved infra, or better documentation.
 
-:::info
+To contribute to this project, please follow the [fork and pull request](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) workflow.
 
-This page may contain outdated information. It will be updated as soon as possible.
+## Prerequisites
```

Comment:
Do these prerequisites apply to both contributing to code and to docs? It's not clear.

Reply:
Good call, split them